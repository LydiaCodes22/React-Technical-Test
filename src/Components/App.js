import React, { useState } from "react";
import "../Styles/app.css";
import Search from "./Search";
import ImageResults from "./ImagesResults";
import getImages from "../Functions/getImages";
// import cors from "cors";


const App = () => {

const [images, setImages] = useState([]);


return (
    <div className="app">
      <img src={"https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"} alt="nasa logo" id="nasaLogo"/>
      <Search  getImages={getImages} setImages={setImages} />
      {images && <ImageResults images={images}/>}
    </div>
  );
}

export default App;