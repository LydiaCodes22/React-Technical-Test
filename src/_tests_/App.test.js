import { render } from '@testing-library/react';
import App from '../Components/App';

describe("App", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});