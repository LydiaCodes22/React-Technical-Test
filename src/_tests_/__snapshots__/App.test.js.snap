// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <img
      alt="nasa logo"
      id="nasaLogo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <div
      class="search"
    >
      <form>
        <input
          placeholder="Search"
          type="text"
          value=""
        />
        <input
          type="submit"
          value="Submit"
        />
      </form>
    </div>
    <div />
  </div>
</DocumentFragment>
`;
